jupyterhub/notebook-images/overlays/build/elyra-notebook-buildconfig.yaml

```
@@ -9,6 +9,13 @@ spec:
     to:
       kind: ImageStreamTag
       name: s2i-lab-elyra:v0.0.7
+  resources:
+    limits:
+      cpu: "2"
+      memory: 4Gi
+    requests:
+      cpu: "1"
+      memory: 2Gi
```

```suggestion
      cpu: "2"
      memory: 8Gi
    requests:
      cpu: "1"
      memory: 6Gi
```

Elyra image are built with jupyterlab build which requires more memory, other wise goes into oom issues.